react-ui-kit/src/Avatar/Avatar.test.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import { Avatar } from '../Avatar';
+import { ReactComponent as PersonOutlineIcon } from '../Icons/system/PersonOutlineIcon.svg';
+import { ReactComponent as AddpersonOutlineIcon } from '../Icons/system/AddpersonOutlineIcon.svg';
+
+jest.mock('../common', () => ({
+  DEFAULT_THEME: {
+    color: {
+      neutral: {
+        white: '',
+      },
+    },
+  },
+}));
+
+describe('Avatar', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const validSrc = 'https://duckduckgo.com/assets/common/dax-logo.svg';
+  const inValidSrc = 'https://duckdusets/common/dax-logo.svg';
```

Comment:
invalid - это одно слово

```suggestion
  const invalidSrc = 'https://duckdusets/common/dax-logo.svg';
```

Reply:
да, ты прав, поправлю